### PR TITLE
UI First Commit

### PR DESCRIPTION
**Pull request by kylercressall**
When you run the UI python file, you can open and run a file. 

TODO: 
- Write to the output field instead of the console
- Be able to read in and submit text as user input

### CODE REVIEW

**Comment by kylercressall** [comment]
will be good to merge after the organizational branch I made is merged

**Review by j-c-williams — APPROVED** [review]
Looks good, thanks for doing this!

**Pull request merged**
